src/common/components/shared/fileName/FileName.scss

```diff
@@ -0,0 +1,11 @@
+.job-filename {
+    display: block;
+    font-weight: bold;
+    padding-top: 20px;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+
+    &__min { max-width: 320px; }
+    &__max { max-width: 75vw; }
```

Comment:
Use `75%` not `75vw` for JobStatus and Settings pages. `75vw` is too wide.
Leave `75vw` for Summary page